imagetagger/imagetagger/images/models.py

```diff
@@ -68,10 +84,61 @@ class Meta:
         default=None
     )
     pinned_by = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='pinned_sets')
+    zip_finished = models.BooleanField(default=True)
```

Comment:
After thinking about it, *finished" might not be the right term; maybe something like "available" or "processing" would be more adequate. After all, the ZIP is never finished if changes are possible at all times ;-)

Reply:
Probably a field like "zip_state" would be more appropriate, for example allowing choices like

* nonexistant
* processing
* ready